src/components/Text/Text.tsx

```diff
@@ -14,14 +14,15 @@ export type Size =
   | 'callout';
 
 export type Variant =
-  | 'error'
-  | 'base'
+  | 'neutralSubtle'
```

Comment:
could this use the `Variant` type from `Heading` also? (or define them in Text and import into Heading)